rust/src/pgsql/pgsql.rs

```diff
@@ -36,17 +36,27 @@ static mut PGSQL_MAX_TX: usize = 1024;
 
 #[repr(u8)]
 #[derive(Copy, Clone, PartialOrd, PartialEq, Eq, Debug)]
-pub enum PgsqlTransactionState {
-    Init = 0,
-    RequestReceived,
-    ResponseDone,
-    FlushedOut,
+pub enum PgsqlTxReqProgress {
+    TxReqInit = 0,
+    TxReqReceived,
+    TxReqDone,
+    TxReqFlushedOut,
+}
+
+#[repr(u8)]
+#[derive(Copy, Clone, PartialOrd, PartialEq, Eq, Debug)]
+pub enum PgsqlTxResProgress {
+    TxResInit = 0,
+    TxResReceived,
+    TxResDone,
+    TxResFlushedOut,
```

Comment:
Values look the same for req and res : could it be one enum ? used by two fields

Reply:
Likely. I had understood that Victor preferred them separated, maybe he'll see this differently when he returns...

Reply:
It does feel nicer to see the enum variant and be able to tell whether it's a request or response, but... ok, that's probably not reason enough.